src/test/java/de/unijena/cheminf/fragment/fingerprint/FragmentFingerprintTest.java

```diff
@@ -25,9 +25,340 @@
 
 package de.unijena.cheminf.fragment.fingerprint;
 
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.openscience.cdk.fingerprint.IBitFingerprint;
+import org.openscience.cdk.fingerprint.ICountFingerprint;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.BitSet;
+import java.util.HashMap;
+import java.util.List;
+
 /**
- *
+ *  Class to test the correct working of FragmentFingerprinter
  */
-public class FragmentFingerprintTest extends FragmentFingerprint{
+public class FragmentFingerprintTest  {
+    //<editor-fold desc="private static class variables" defaultstate="collapsed">
+    /**
+     * Is a list in which all molecule fragments are stored that are read in from the CSV file.
+     */
+    private static ArrayList<HashMap<String, Integer>> moleculeFragmentList = new ArrayList<>(); // TODO local variable if only test the last molecule
+    /**
+     * Is a list that contains all fragments, the fingerprint is then generated based on these fragments.
+     */
+    private static ArrayList<String> fragmentList = new ArrayList<>();
+    /**
+     *  fragmentFingerprinter
```

Comment:
Extra space (check whole repository, is not mentioned further)
**F**ragmentFingerprinter